packages/diffhtml-rust-parser/src/parser.rs

```diff
@@ -0,0 +1,97 @@
+//extern crate wee_alloc;
+
+use tl::{Node, Parser};
+use serde_wasm_bindgen::to_value;
+use wasm_bindgen::prelude::*;
+use wasm_bindgen::{intern};
+use js_sys::{Reflect, JsString};
+use js_sys::Array;
+use js_sys::Object;
+
+// Opt for a smaller allocator to save on resources.
+//#[global_allocator]
```

Comment:
Using wee_alloc shaves off ~300bytes from the gzip output, but is much slower. I'd rather have the speed for now, and then add this back once the performance has been addressed.